nix/bootstrap.nix

```diff
@@ -1,12 +1,12 @@
 { debug ? false, stage0debug ? false, extraCMakeFlags ? [],
-  stdenv, lib, cmake, gmp, gnumake, bash, buildLeanPackage, writeShellScriptBin, runCommand, symlinkJoin, lndir, perl, gnused, darwin, llvmPackages_14,
+  stdenv, lib, cmake, gmp, gnumake, bash, buildLeanPackage, writeShellScriptBin, runCommand, symlinkJoin, lndir, perl, gnused, darwin, llvmPackages,
```

Comment:
Where is the version of LLVM setup? Does it pick it up from:

```
# nix/packages.nix
llvmPackages = if stdenv.isDarwin then llvmPackages_11 else llvmPackages_14;

```

[`nix/packages.nix:8](https://github.com/leanprover/lean4/blob/3134e5c55d714151927fd16a6c7e61fe8dab8d63/nix/packages.nix#L8?)

Reply:
yes